extra/1-radio-stations.js

```diff
@@ -15,6 +15,14 @@
 
 // `getAllFrequencies` goes here
 
+function getAllFrequencies () {
+  let tmpFreq = [];
+  for (i=87; i <= 108; i++) {
```

Comment:
It looks like you're using i here without declaring it. It's a good idea to use let here, this will limit the scope of the variable to the for loop. We will discuss more about variable scopes in the JS3 module.

Reply:
Yes, I see. I just forgot about it, I didn't do it for purpose.